app/controllers/lti_registration_wips_controller.rb

```diff
@@ -0,0 +1,103 @@
+class LtiRegistrationWipsController < ApplicationController
```

Comment:
Not clear what is handled by this controller. Some class comments are necessary.